accessibility_api/accessibility_lib/utils/WinUtil.py

```diff
@@ -0,0 +1,199 @@
+'''
```

Comment:
OK so I see what window utils does now. I think I would strongly prefer for it to be as stateless as possible. And even though it's not super efficient, to not store anything we saw. Only use the utils to get what we want. If you still want to store something, it should really be outside of the utils. (especially stuff related to target search).